appservice/staticsite/README.md

```diff
@@ -17,7 +17,7 @@ These code samples will show you how to manage Web using Azure SDK for Golang.
 This project framework provides examples for the following services:
 
 ### Web
-* Using the Azure SDK for Golang - Web Management Library [web/armappservice](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/web/armappservice) for the [Azure Web Manager API](https://docs.microsoft.com/en-us/rest/api/web-apps/)
+* Using the Azure SDK for Golang - Web Management Library [appservice/armappservice](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appservice/armappservice) for the [Azure Web Manager API](https://docs.microsoft.com/en-us/rest/api/web-apps/)
```

Comment:
The rest api link is invalid, it should be: https://docs.microsoft.com/en-us/rest/api/appservice/
```suggestion
* Using the Azure SDK for Golang - Web Management Library [appservice/armappservice](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appservice/armappservice) for the [Azure AppService Manager API](https://docs.microsoft.com/en-us/rest/api/appservice/)
```